omero/sysadmins/unix/install-web/install-wsgi.txt

```diff
@@ -89,6 +89,18 @@ Start the Gunicorn worker processes running one thread listening on 127.0.0.1:40
     ... static files copied to '/usr/local/dev/openmicroscopy/dist/lib/python/omeroweb/static'.
     Starting OMERO.web... [OK]
 
+.. note::
+    Additional setting can be configured from command line arguments:
+
+    * --workers WORKERS - The number of worker processes for handling requests.
```

Comment:
Also, the double `--` are not formatting on staging, this needs to be marked up with back ticks